packages/popover/src/Popover.tsx

```diff
@@ -203,7 +217,11 @@ function Popover({
   `;
 
   const Root = usePortal ? Portal : Fragment;
-  const rootProps = usePortal ? { className: portalClassName } : {};
+  const rootProps = usePortal
+    ? portalContainer
+      ? { container: portalContainer }
+      : { className: portalClassName ?? undefined }
```

Comment:
do we need to explicitly write undefined here? if portalClassName is undefined won't it evaluate as such without needing to provide a fallback

Reply:
I believe this was to make TS happy.